rholang/src/test/scala/coop/rchain/rholang/interpreter/PrettyPrinterTest.scala

```diff
@@ -745,6 +745,18 @@ class ProcPrinterSpec extends FlatSpec with Matchers {
         |}""".stripMargin
   }
 
+  it should "Print receive in curly brackets" in {
+    checkRoundTrip(
+      """match Nil {
+        |  _ => {
+        |    for( @{_} <- @{Nil} ) {
+        |      Nil
+        |    }
+        |  }
+        |}""".stripMargin
```

Comment:
We should actually always put braces around the process that is the body of a match case

Reply:
Except for sends, right?

Reply:
And out of interest: why do you think so?

Reply:
why except for sends?
I think so because of consistency :)

Reply:
And implementation ease

Reply:
and then, maintainance ease (I really hope we'll rewrite the pretty-printer in the future)

Reply:
Nevermind, it was late. I somehow thought you were talking about the grammar and not the PrettyPrinter.